salt/search/elasticsearch.sls

```diff
@@ -45,3 +45,12 @@ elasticsearch-config:
         - group: elasticsearch
         - mode: 644
         - template: jinja
+        - require:
+            - pkg: elasticsearch
+
+elasticsearch-ready:
```

Comment:
I encountered this from time to time
We are using this polling in multiple places, we will extract it into a shell script in builder-base-formula